cmd/lock-rpc-server_test.go

```diff
@@ -439,6 +449,12 @@ func TestLockServers(t *testing.T) {
 	if runtime.GOOS == "windows" {
 		return
 	}
+
+	currentIsDistXL := globalIsDistXL
+	defer func() {
+		globalIsDistXL = currentIsDistXL
+	}()
+
```

Comment:
Pass ``currentIsDistXL`` as argument to the anonymous function to avoid any runtime surprises.